code/modules/reagents/Chemistry-Reagents/Other-Reagents.dm

```diff
@@ -26,7 +26,7 @@
 		src.data["cloneable"] = 0 //On mix, consider the genetic sampling unviable for pod cloning, or else we won't know who's even getting cloned, etc
 		if(src.data["viruses"] || data["viruses"])
 
-			var/list/mix1 = data["viruses"]
+			var/list/mix1 = src.data["viruses"]
```

Comment:
I don't like this fix, rename the argument in this proc from ```data``` to something like ```mix_data```
because the code is very annoying to look at seeing ```src.data``` and ```data``` when in most contexts, those appear to mean the same thing!